features/keychain/module/crypto/schnorr-z.js

```diff
@@ -0,0 +1,63 @@
+import { hashSync } from '@exodus/crypto/hash'
+import { hmacSync } from '@exodus/crypto/hmac'
+import { randomBytes } from '@exodus/crypto/randomBytes'
+import * as secp256k1 from '@noble/secp256k1'
```

Comment:
Yes. We can directly import `@exodus/keychain/module/crypto/secp256k1` for use in the library.

Reply:
Unless you have a different suggestion?

Reply:
Maybe in the future, an asset won't be able/won't need to define signing, signing is always happing via the keychain.  (move funds/import key would break this logic though)

Reply:
We would need to add support for adding a private key to the keychain. Not sure what that would look like. Currently the keychain operates by deriving private keys from a seed. This private key would not be associated with a seed.